test/models/github_service_test.rb

```diff
@@ -0,0 +1,48 @@
+require './test/test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.